<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="QGroupBox_live_view">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1341</width>
      <height>611</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1341</width>
      <height>611</height>
     </size>
    </property>
    <property name="title">
     <string>Live View</string>
    </property>
    <widget class="ImageView" name="image_view" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1311</width>
       <height>521</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1311</width>
       <height>521</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_live_view">
     <property name="geometry">
      <rect>
       <x>1170</x>
       <y>570</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string>Stop Live View</string>
     </property>
    </widget>
    <widget class="QLabel" name="polling_label">
     <property name="geometry">
      <rect>
       <x>236</x>
       <y>563</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Polling rate (Hz)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="polling_frequency">
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>582</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="pv_prefix_label">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>563</y>
       <width>59</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>59</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>PV Prefix:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_live_view">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>570</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>103</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string>Start Live View</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pv_prefix">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>582</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>dp-ADSim:Pva1:Image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="plotting_frequency">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>580</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>211</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="plotting_label">
     <property name="geometry">
      <rect>
       <x>457</x>
       <y>561</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Plotting rate (Hz)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="log_image">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>580</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>logn image</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="QGroupBox_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>1341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QPlainTextEdit" name="log_plain_text_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1321</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
